src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx

```diff
@@ -29,7 +29,7 @@ export default function AdminLinks(props: WalletState) {
 
   return (
     <div className="grid p-4 gap-3 border-b border-prim">
-      <h3 className="font-heading font-bold text-sm text-gray-d1 dark:text-gray">
```

Comment:
Need to remove `font-heading` and `font-bold` from these components as well:
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/ConnectedWallet/Details/MobileTitle.tsx#L8
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/KadoModal.tsx#L38
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/ConnectedWallet/Details/Favourites/index.tsx#L16
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/components/WalletSuite/WalletSelectorOpener/WalletModal.tsx#L15
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/contexts/ErrorContext/InstallWalletContent.tsx#L25
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/ui-fixes/src/pages/Admin/Charity/WidgetConfigurer/WidgetConfigurer.tsx#L7

Reply:
i will do it

Reply:
@misicnenad Done, thanks for finding these file , i mainly used regular expressions to find <h> tags with these classnames